playbooks/rhv/inventory.example

```diff
@@ -0,0 +1,99 @@
+[all:vars]
+app_dns_prefix=apps
+public_hosted_zone=example.com
+load_balancer_hostname=lb0.{{public_hosted_zone}}
+openshift_master_cluster_hostname="{{ load_balancer_hostname }}"
+openshift_master_cluster_public_hostname=openshift-master.{{ public_hosted_zone }}
+openshift_master_default_subdomain="{{ app_dns_prefix }}.{{ public_hosted_zone }}"
+openshift_public_hostname="{{openshift_master_cluster_public_hostname}}"
+
+[OSEv3:children]
+nodes
+masters
+etcd
+lb
+
+[OSEv3:vars]
+# General variables
+ansible_ssh_user=root
+console_port=8443
+debug_level=2
+deployment_type=openshift-enterprise
+openshift_debug_level="{{ debug_level }}"
+openshift_deployment_type="{{ deployment_type }}"
+openshift_master_cluster_method=native
+openshift_node_debug_level="{{ node_debug_level | default(debug_level, true) }}"
+openshift_release='3.10'
+openshift_vers=v3_10
+
+# RHV Specific Settings
+openshift_enable_service_catalog=False
+# openshift_cloudprovider_kind=ovirt
+# openshift_cloudprovider_ovirt_auth_url=
+# openshift_cloudprovider_ovirt_username=
+# openshift_cloudprovider_ovirt_password=
+
+# Pod Networking
+os_sdn_network_plugin_name=redhat/openshift-ovs-networkpolicy
+
+# Registry
+openshift_hosted_registry_replicas=1
+openshift_hosted_registry_storage_kind=nfs
+openshift_hosted_registry_storage_access_modes=['ReadWriteMany']
+openshift_hosted_registry_storage_host=
```

Comment:
Should be openshift_hosted_registry_storage_host=nfs..{{public_hosted_zone}} no?

Reply:
Except for this parameter why are the other required here ? Why not refer to the https://github.com/openshift/openshift-ansible/blob/master/inventory/hosts.example for all the defaults and options? Its confusing...

Reply:
This is included in error... I had removed the hosted registry lines at one point, but may have lost those changes. I'll take another look at streamlining the example inventory.

Reply:
+1